src/network/networkctl.c

```diff
@@ -202,6 +213,32 @@ static int decode_link(sd_netlink_message *m, LinkInfo *info, char **patterns) {
         else if (sd_netlink_message_read(m, IFLA_STATS, sizeof info->stats, &info->stats) >= 0)
                 info->has_stats = true;
 
+        /* fill kind info */
+        r = sd_netlink_message_enter_container(m, IFLA_LINKINFO);
+        if (r >= 0) {
+                r = sd_netlink_message_read_string(m, IFLA_INFO_KIND, &received_kind);
+                if (r >= 0) {
+                        r = sd_netlink_message_enter_container(m, IFLA_INFO_DATA);
+                        if (r >= 0) {
+                                if (streq(received_kind, "bridge")) {
+                                        (void) sd_netlink_message_read_u32(m, IFLA_BR_FORWARD_DELAY, &info->forward_delay);
```

Comment:
Horrible indentation :-) It is better to move them to, say, `decode_bridge_attrs()`.

Reply:
Hahaha Well I am having fun with this will make a separate func and add them all. thanks